build-lualib.js

```diff
@@ -1,17 +1,11 @@
 require("ts-node/register/transpile-only");
```

Comment:
I think we can actually get rid of this file. The reason for using this was because we couldn't just use command line tstl because we needed to write the bundle. That part is now gone, so we can just use tstl CLI in our build script.

Reply:
Done, now invoking dist/tstl.js from package.json

Reply:
Currently not possible due to unknown issue tracked in  #1232